src/k8s.rs

```diff
@@ -100,7 +101,9 @@ pub async fn stream_single_pod_logs(
 
     while let Some(line) = logs.try_next().await? {
         let pretty_pod_name = &pod.name_any().truecolor(color.r, color.g, color.b);
-        println!("{} {}", pretty_pod_name, line);
+        if (filter.len() > 0 && line.contains(filter)) || filter.len() == 0 {
```

Comment:
🚫 **[clippy]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> 
<pre><code>error: length comparison to zero
   --> src/k8s.rs:104:13
    |
104 |         if (filter.len() > 0 && line.contains(filter)) || filter.len() == 0 {
    |             ^^^^^^^^^^^^^^^^ help: using `!is_empty` is clearer and more explicit: `!filter.is_empty()`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_zero
    = note: `-D clippy::len-zero` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::len_zero)]`
</code></pre>
<!-- __reviewdog__:ChBmZGJkOWZhY2EwYWE3MzFmEgZjbGlwcHk= -->